lighthouse-service/main.go

```diff
@@ -2,15 +2,13 @@ package main
 
 import (
 	"context"
-	keptnapi "github.com/keptn/go-utils/pkg/api/utils"
-	"log"
-	"os"
-
 	cloudevents "github.com/cloudevents/sdk-go/v2"
 	"github.com/kelseyhightower/envconfig"
-
+	keptnapi "github.com/keptn/go-utils/pkg/api/utils"
 	keptncommon "github.com/keptn/go-utils/pkg/lib/keptn"
 	"github.com/keptn/keptn/lighthouse-service/event_handler"
+	"log"
+	"os"
 )
```

Comment:
We need to define some cross-IDE format like we have in [Bridge](https://github.com/keptn/keptn/blob/master/bridge/.editorconfig)